src/react-app-env.d.ts

```diff
@@ -4,6 +4,7 @@
 declare namespace NodeJS {
     interface ProcessEnv {
         REACT_APP_SERVER: string;
+        REACT_APP_PROPUBLICA_API_KEY: string;
```

Comment:
This is just a note really.  In the future we will be using our backend to make the requests.  If we put the API key on the front end then anybody who uses our app will have our API key.  That will probably be your next task.

Reply:
Got it